src/blocks/social-profiles/inspector.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * External dependencies
+ */
+import includes from 'lodash/includes';
+import escape from 'lodash/escape';
```

Comment:
I believe you can also do this:

 `import { includes, escape } from 'lodash';`

Reply:
This has been done.